### PR TITLE
cemu: init at 1.3

### PR DESCRIPTION
**Pull request by luc65r**
###### Motivation for this change

Added cemu app

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -4241,6 +4241,11 @@
     github = "ltavard";
     name = "Laure Tavard";
   };
+  luc65r = {
+    email = "lucas@ransan.tk";
+    github = "luc65r";
```
Comment (mmahut) [inline maintainers/maintainer-list.nix]: Please add your ```githubId```.

**Comment by mmahut** [comment]
@GrahamcOfBorg build cemu

**Comment by aanderse** [comment]
So there are two emulators called (cemu)[http://cemu.info/]?

**Review by doronbehar — APPROVED** [review]

**Comment by luc65r** [comment]
> So there are two emulators called [cemu](http://cemu.info/)?

Yes...

**Comment by aanderse** [comment]
Has anyone tested this? Seems like an easy merge. @luc65r ping me back in a few days of no one else reports back on this and I'll test+merge it. Make sense to you @mmahut?

**Comment by luc65r** [comment]
@aanderse It seems like no one is interested in testing and merging it. If you could do it, it would be nice, but it isn't urgent.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -4241,6 +4241,12 @@
...
+    githubId = 59375051;
+    name = "Lucas Ransan";
+  };
```
Comment (timokau) [inline maintainers/maintainer-list.nix]: Welcome!

**Review by timokau — APPROVED** [review]

**Pull request merged**